spec/source_spec.rb

```diff
@@ -0,0 +1,17 @@
+require_relative '../classes/source'
+require_relative '../classes/item'
+
+describe Source do
+  let(:name) { 'Motion Pictures' }
+
+  subject { described_class.new(name) }
+
+  describe '#add_item' do
+    it 'adds an item to the source and sets source for the item' do
+      item = Item.new('2005/07/15')
+      subject.add_item(item)
+      expect(subject.items).to include(item)
+      expect(item.source).to eq(subject)
+    end
```

Comment:
- [ ] Great job working on the requested changes but I think it would be great if you fix the source issue so when I press on list sources I can still not see all the sources although I added another source so please as mentioned with the previous reviewer solve the issue 🎉🙏🏻 

- [ ]  I realized when I check the list of movies I get the publish and source details added, but when I tried to check the list of sources, and didn't get any of the added data to display. I believe this is a bug in your project and I would advice you to fix it

- Although I added a new source but it I can't see it displayed there .🔥✅
![image](https://github.com/2004-okari/CATALOG/assets/58553711/14db0916-34d9-4a0d-9ed6-d3d2c08cd857)